simplekv/__init__.py

```diff
@@ -181,6 +182,8 @@ def _check_valid_key(self, key):
 
         :param key: The key to be checked
         """
+        if not isinstance(key, text_type) and key is not None:
```

Comment:
@StephanErb you are completly right. I would have betted that I added the check only because of failing tests otherwise, but I can't reproduce any test failures anymore. Thus, I removed that part of the check.